### PR TITLE
Check lat/long are actual numbers

### PR DESCRIPTION
**Pull request by cynngah**
closes #209

---
This change is [Reviewable](https://reviewable.io/reviews/fgpv-vpgf/geoapi/110)

### CODE REVIEW

**Comment by james-rae** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*

**Comment by james-rae** [comment]
Reviewed 2 of 4 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*

**Comment by alyec** [comment]
Reviewed 2 of 3 files at r1, 2 of 4 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[src/layer.js, line 938 \[r3\]](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110#-KHKvPeT0pf9NqM1tJ2H:-KHKvPeT0pf9NqM1tJ2I:698138002) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/8fd59ebe69b1a854c7df796b02efab9cb9c8740f/src/layer.js#L938)):*
> ```JavaScript
> 
> /**
> * Given csv array, check whether or not specified indices for lat/long are actually numeric
> ```

Can you reword this to make it more general (e.g. `Given a 2d array in column x row format, check if all entries in the two given columns are numeric`)?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*

**Comment by cynngah** [comment]
Review status: 4 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/layer.js, line 938 \[r3\]](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110#-KHKvPeT0pf9NqM1tJ2H:-KHKwiEEF49ZtacvopdV:-499146009) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/8fd59ebe69b1a854c7df796b02efab9cb9c8740f/src/layer.js#L938)):*
<details><summary><i>Previously, alyec (Aly Merchant) wrote…</i></summary>
> Can you reword this to make it more general (e.g. `Given a 2d array in column x row format, check if all entries in the two given columns are numeric`)?</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*

**Comment by james-rae** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*

**Comment by alyec** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/fgpv-vpgf/geoapi/110)*
